lib/Traits/TraitsClasses/TimeVaryingTraits.ts

```diff
@@ -8,7 +8,7 @@ export default class TimeVaryingTraits extends mixTraits(CatalogMemberTraits) {
     description: "The current time at which to show this dataset.",
     type: "string"
   })
-  currentTime?: string;
+  currentTime?: string | null;
```

Comment:
does this trait need to set the isNullable field to true? Also, why do we have a separate isNullable field in the trait system?

Reply:
You are right! 

I think `isNullable` is used only during [de-serialization](https://github.com/TerriaJS/terriajs/blob/0e5ff1bd512597808274ab58805d5d1ce6b75acf/lib/Traits/Decorators/primitiveTrait.ts#L55-L65) of a model from json. And that is because we don't have the type information during runtime.

There is low chance that a json serialized model will have `null` as `currentTime`, but I'll enable that anyways.